pkgs/development/node-packages/node-packages.json

```diff
@@ -168,8 +168,6 @@
 , "prebuild-install"
 , "prettier"
 , "prettier-plugin-toml"
-, "prisma"
-, "@prisma/language-server"
```

Comment:
Why was `@prisma/language-server` removed? It's not part of the https://github.com/prisma/prisma monorepo and is not packaged in this PR.

Reply:
I wonder how useful it is when the version differs to the `prisma` package. And I couldn't find any references to it so it's not used by anything within nixpkgs. Though I guess it would be less controversial to keep it.

Reply:
> I wonder how useful it is when the version differs to the prisma package.

It's not optimal but it's mostly fine, it's only really a problem when there are new features that affect the Prisma schema language and that doesn't happen too often lately. But we should definitely extract it to a normal package and update it regularly.

> And I couldn't find any references to it so it's not used by anything within nixpkgs.

It's used by the users: https://github.com/search?type=code&q=%40prisma%2Flanguage-server+language%3ANix&l=Nix